src/modules.py

```diff
@@ -10,22 +10,23 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+print(sys.argv)
```

Comment:
You're off to a good start Hunter, but pay a little closer attention to what you're being asked to do. As it stands your current code will simply print the current list of arguments all on one line. However, If you want to print the arguments line by line you need to iterate through the list with a loop:
`for x in sys.argv:
	print(x)`